tools/print-compiled-keymap.c

```diff
@@ -53,32 +57,64 @@ main(int argc, char *argv[])
 
     keymap_path = argv[optind];
 
-    ctx = test_get_context(0);
+    ctx = xkb_context_new(XKB_CONTEXT_NO_FLAGS);
     if (!ctx) {
         fprintf(stderr, "Couldn't create xkb context\n");
-        goto err_out;
+        goto out;
     }
 
-    keymap = test_compile_file(ctx, keymap_path);
+    if (streq(keymap_path, "-")) {
+        FILE *tmp;
```

Comment:
Heh stdio is pretty painful. I do think we need to read in a loop though, assuming the the stdin input is at most 4096 bytes is not great.

Reply:
huh, I thought I had done that. Sorry, will fix.